apiary.apib

```diff
@@ -374,6 +374,8 @@ We list all backwards-compatible additions here. These are currently available i
 
 #### Latest
 
+- The task assignee is now nullable.
```

Comment:
Q: Maybe it would make sense to add a small explanation? 

> The task assignee is now nullable (unassigned task).

Reply:
How about `The task assignee is now nullable: you can create / update tasks without an assignee by passing null as assignee. The tasks.list and tasks.info endpoints also return null as assignee in case the task is unassigned. On API versions older than 2019-10-09, the assignee is returned as  { type: user, id: null } in case the task is unassigned.`